godtools/App/Share/Data/OptInOnboardingBannerEnabledRepository/OptInOnboardingBannerEnabledRepository.swift

```diff
@@ -7,16 +7,32 @@
 //
 
 import Foundation
+import Combine
 
 class OptInOnboardingBannerEnabledRepository {
+        
+    private static let sharedPublishers: OptInOnboardingBannerEnabledRepositoryPublishers = OptInOnboardingBannerEnabledRepositoryPublishers()
     
-    let userDefaultKey = UserDefaultKeys.optInOnboardingBannerDisabled
+    private let cache: OptInOnboardingBannerEnabledCache
```

Comment:
It seems kinda overkill to me to have a whole cache for 1 value, and then a repository on top of that.  But I guess, in general, repositories would be pulling the data from a separate class anyway, not doing it themselves.  So maybe it fits the pattern better to have it separated out.  Just feels a little contrived in this case 😄

Reply:
Yeah it does seem that way to me too, a lot of work for cacheing a single attribute.  I'm hoping that sticking to a consistent pattern will help when it comes to additional developers working in the code-base.